plugins/ingest-attachment/build.gradle

```diff
@@ -81,9 +81,6 @@ dependencies {
   api "org.apache.pdfbox:fontbox:${versions.pdfbox}"
   api "org.apache.pdfbox:jempbox:1.8.17"
   api "commons-logging:commons-logging:${versions.commonslogging}"
```

Comment:
Can you confirm that this plugin still works as expected?
Originally Elasticsearch claimed the was not compatible with FIPS mode https://www.elastic.co/guide/en/elasticsearch/reference/7.17/fips-140-compliance.html#_limitations_10

That may have changed for them ( https://github.com/elastic/elasticsearch/issues/88118#issuecomment-1170499326 ) but it would be good to be sure regarding OpenSearch

Reply:
Almost forgot about this limitation, thanks for exposing it. Let me comment on it when test coverage is done.

Reply:
Related to this, there appears to be new code coming in for 2.16 that references algorithms that are not FIPS-complaint and may also need to be addressed https://github.com/opensearch-project/OpenSearch/blob/main/modules/ingest-common/src/main/java/org/opensearch/ingest/common/FingerprintProcessor.java#L75

Reply:
Additionally it seems likely that BC dependencies were added to support encrypted PDF's. But current [tests](https://github.com/opensearch-project/OpenSearch/blob/548a6505d6eaa88170462ef5eaf45a89a367126e/plugins/ingest-attachment/src/test/java/org/opensearch/ingest/attachment/AttachmentProcessorTests.java#L207) assert otherwise.